Communicate iOS sensors won't be auto enabled from 2025.5 (#1199)
home-assistant/companion.home-assistant: docs/core/sensors.md

On macOS, sensors update in the same situations as above as well as immediately when some sensors change.

### ![iOS](/assets/iOS.svg) From 2025.5

> Not all ![iOS](/assets/iOS.svg) sensors are enabled by default, if you don't see the sensor you want in Home Assistant, reach to Companion App Settings -> Sensors, and enable the desired one.

Reviewer: ```suggestion
Not all ![iOS](/assets/iOS.svg) sensors are enabled by default. If you don't see your sensor in Home Assistant, go to **Companion App Settings** > **Sensors**, and enable it manually.
```
